### ISSUE TITLE
beA: Spalten-Reihenfolge selber sortieren können

### ISSUE THREAD

**Issue by stm9x9** [jlawyerorg/j-lawyer-org#380]

**Comment by j-dimension** [comment]
Du beschreibst eine Lösung, und ich würde gern das Problem verstehen wollen.

Ist die Reihenfolge nicht optimal? --> dann könnte ich sie anpassen
Gibt es ein Anwendungsszenario wo man wirklich selbst definieren muss?

Jedes mehr an Flexibilität kostet: initialer Entwicklungsaufwand, zusätzliche Fehlerbehandlungen, Wartung durch komplexeren Code. Daher würde ich gern erst besser verstehen ob Kosten/Nutzen in sinnvollem Verhältnis stehen.

**Comment by stm9x9** [comment]
Ich kenne es nur aus der beA-web so. Da gibt es bei jedem Anwender bei uns sehr individuelle Vorstellungen, welche Spalte / welche Information man vorne haben will. Wobei ich noch nicht weiß, ob/wie sich das in Zukunft geänderte Handling auswirkt. Momentan wird beA bei uns auch noch als Archiv verwendet und es ist einfach sehr viel drin. Wenn in Zukunft die beAs direkt in die Akte genommen werden, dann wird beA ja deutlich leerer sein. 
Ich würde es daher mal als Thema für 2.0 vorschlagen. 
Ich hatte halt gehofft, Du nimmst da irgendeine Listendarstellungsbibliothek zum Anzeigen und man müßte das Spalten verschieben einfach nur aktivieren.

**Comment by Dickmaulruessler** [comment]
Dies betrifft bea und E-Mail und sollte sicherlich einheitlich behandelt werden. 

Auch ich würde mir eine andere Spalten-Reihenfolge wünschen: 
Datum - Absender -Betreff - ...

In Thunderbird gibt es aber standardmäßig eben die aktuell verwendete Reihenfolge. Daher wird es wohl einige geben, die das so beibehalten möchten. Ich finde den "Betreff" am Anfang aber unlogisch. 

Am einfachsten wäre es, wenn jeder die Spalten so schieben kann, wie er möchte. Inwieweit die Umsetzung aufwändig ist, kann ich nicht beurteilen. 

Falls zu aufwändig, sollten wir uns hier auf eine allgemein aktzeptiere Reihenfolge verständigen. 
Jedenfalls das Betreff am Anfang (lang, häufig nicht aussagekräftig genug) macht auch aus meiner Sicht nicht so viel Sinn und kostet dort wo man zuerst hinguckt eben viel Platz.

**Comment by j-dimension** [comment]
> Datum - Absender -Betreff - ...

Damit könnten sicherlich sehr viele Anwender gut arbeiten.

**Comment by stm9x9** [comment]
Ich würde auf jeden Fall das Az Empfänger weiter vorne ansiedeln. Wenn da Post für mehrere Anwälte aufläuft, dann werden die meist an dem eigenen Az sehen können, was für welchen Anwalt ist.

**Comment by Dickmaulruessler** [comment]
> Du beschreibst eine Lösung, und ich würde gern das Problem verstehen wollen.
> 
> Ist die Reihenfolge nicht optimal? --> dann könnte ich sie anpassen Gibt es ein Anwendungsszenario wo man wirklich selbst definieren muss?

Ich würde das argumentativ gerne noch einmal ergänzen. 

Die jetzige Reihenfolge mit dem Betreff am Anfang ist glaube ich für jeden (nicht-Thunderbird-Nutzer) sehr gewöhnungsbedürftig. Logisch (und für Outlook-Nutzer wohl standard) ist für mich nur **Datum-Absender-Betreff.**

Bei j-lawyer unter **beA** ist die Reihenfolge gewöhnungsbedürftig. Dort bleiben aber **alle** Spalten dennoch **sichtbar**. Damit kann man klarkommen. 
Unter E-Mail sieht man aber **nur die zwei** Spalten Betreff und Absender. Für das Datum muss man rechts rausscrollen. 

![jfkldöfjklsdöa](https://user-images.githubusercontent.com/46451749/188468563-fd3600fc-568e-4077-9ad1-969c1d2d505b.jpg)

Wenn man bei den E-Mails die Spalten dann manuell verändert, ist alles wieder gut. Man sieht dann jede Spalte. Diese Einstellung wird aber leider nicht gespeichert. 

Falls unter E-Mail mit einer Übergangslösung also die individuelle Spaltenbreite gespeichert werden würde, so dass man sich auch dauerhaft die Datumsspalte stets sichtbar einstellen kann, wäre schon viel gewonnen. 

Soll ich für E-Mail ein gesondertes Ticket aufmachen?

**Comment by j-dimension** [comment]
> Soll ich für E-Mail ein gesondertes Ticket aufmachen?

Nein, das würde ich gemeinsam bearbeiten. Danke für's Nachfragen!
